python/Ganga/__init__.py

```diff
@@ -309,7 +309,7 @@ def getLCGRootPath():
                  filter=Ganga.Utility.Config.expandvars)
 
 lcg_config.addOption(
-    'GLITE_ENABLE', True, 'Enables/disables the support of the GLITE middleware')
+    'GLITE_ENABLE', False, 'Enables/disables the support of the GLITE middleware')
```

Comment:
At the very least the Durham IPPP group as they submit to ARC. Beyond that I believe SNO+ are also still submitting to LCG/CREAM/ARC though I may be wrong in that.